### PR TITLE
Fixed a mistake in the README documentation

### PR DESCRIPTION
**Pull request by alcfeoh**
The code wouldn't work as-is. I renamed the variable so that code would work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ import React from 'react';
 import { injectMocks, extractScenarioFromLocation } from 'data-mocks';
 import gql from 'graphql-tag';
 
-const mocks: Scenarios = {
```
Comment (alcfeoh) [inline README.md]: This type doesn't get imported. Also, this would work with TypeScript only.

File: README.md [hunk 2]
```diff
@@ -173,7 +173,7 @@ const mocks: Scenarios = {
   ]
 };
 
-injectMocks(scenarios, extractScenarioFromLocation(window.location));
```
Comment (alcfeoh) [inline README.md]: Here the variable name didn't exist. It was supposed to be ```mocks``` instead.

**Comment by grug** [comment]
Hey Alain!

Thanks so much for submitting a PR - big fan of your talks!

The changes look good - thanks for spotting those issues. 

I was just wondering if I could get your thoughts on data-mocks and what sorts of things you'd like to see added/changed?

-Dave

**Comment by alcfeoh** [comment]
Thanks! 
I really like the simplicity of `data-mocks`, which got me to blog about it as I think that `data-mocks` is easier to use than Angular mocks for HTTP requests: https://blog.angulartraining.com/using-data-mocks-with-angular-ca481bde3ed0

That's how I found that issue and decided to fix it right away :-)

I don't see anything to add. Leonardo da Vinci once said: "Simplicity is the ultimate sophistication". I agree with that quote.

**Comment by grug** [comment]
Thanks! If you ever think of any feedback, you can always drop me a line :)

**Pull request merged**